spec/controllers/teams_controller_spec.rb

```diff
@@ -184,7 +193,28 @@
 
             expect(Role.last.user).to eql existing_user
           end
+        end
 
+      #TODO: refactor this test
+      context 'selecting the conference options' do
+        let(:conference_1) { FactoryGirl.create(:conference, :in_current_season)}
+        let(:conference_2) { FactoryGirl.create(:conference, :in_current_season)}
+        let(:team_params) do
+          build(:team).attributes.merge(:attendances_attributes=>{
+            '0'=>{
+              option: 1, conference_id: conference_1.id
+            },
+            '1'=>{
+              option: 2, conference_id: conference_2.id
+            }
+          })
+        end
+
+          it 'team student user' do
```

Comment:
Please come up with a more meaningful description for the test and fix the intendation - it makes it kind of hard to read for someone new to it.